src/dispatcher.cc

```diff
@@ -162,6 +163,53 @@ int Dispatcher::Getaddrinfo(const char* node, const char* service, const addrinf
   return 0;
 }
 
+int Dispatcher::Accept4(int sockfd, sockaddr* addr, socklen_t* addrlen, int flags) {
+  const int fd = raw_->Accept4(sockfd, addr, addrlen, flags);
```

Comment:
accept may be called with nullptr for addr and addrlen. So we need to declare and pass our own variables here and copy the values if caller did not pass nullptr. Either fix or add TODO